### PR TITLE
chore: Config travis

### PR DESCRIPTION
**Pull request by drazik**
I used the same configuration as cozy-notes.

REGISTRY_TOKEN is in password-store-team under
`registry/spaces/default/passwords.token`.

GITHUB_TOKEN has been generated with cozy-bot account.

### CODE REVIEW

**Comment by Crash--** [comment]
We should not use the one from Notes since we want to change it. 

We should use downcloud to store our artefacts and not using a github branch to do that.

**Comment by drazik** [comment]
Oh, I thought you did the changes we wanted to do already. I'm going to update the config to use downcloud, then

**Comment by drazik** [comment]
@Crash-- I did the change to use downcloud

File: .travis.yml [hunk 1]
```diff
@@ -1,60 +1,38 @@
...
+  directories:
+  - node_modules
+script:
+  - yarn lint
+  - yarn build
+deploy:
+  - provider: script
+    repo: cozy/cozy-passwords
+    skip-cleanup: true
+    # deploy the build on a build branch and publish to the Cozy registry
```
Comment (Crash--) [inline .travis.yml]:
```suggestion
    # deploy the build on a downcloud and publish to the Cozy registry
```

**Review by Crash-- — APPROVED** [review]
Amazing!

**Pull request merged**